modules/items/client/config/items.client.menus.js

```diff
@@ -21,15 +21,5 @@
       state: 'items.list',
       roles: ['admin', 'superta', 'ta', 'technician']
     });
-    menuService.addSubMenuItem('topbar', 'items', {
```

Comment:
how are we going to access these pages?

Reply:
We totally are, but not from the TA perspective, only the admin is entitled access per the answer to our questioning.